recipes/waf/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "waf" %}
+{% set version = "1.8.21" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/waf-project/waf/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: 4007f38b5a0de2f0def758e30e0f66b0c3d7be0f7b832299e78b514513f3b013
+
+build:
+  number: 0
+  skip: true  # [win]
+  script:
+    - python waf-light --make-waf --tool=swig,cython,boost,use_config
+    - cp waf "${PREFIX}/bin"     # [unix]
+    - copy waf "%LIBRARY_BIN%"   # [win]
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  commands:
+    - waf --help
```

Comment:
This fails on Windows. I'm totally baffled and am completely uninterested in debugging it ATM. Will raise an issue/PR in the feedstock to explain/demonstrate the problem.